dimod/embedding/utils.py

```diff
@@ -192,17 +193,20 @@ def chain_break_frequency(samples, embedding):
         if samples.variable_labels is not None:
             label_to_idx = samples.label_to_idx
             embedding = {v: {label_to_idx[u] for u in chain} for v, chain in iteritems(embedding)}
-        samples = samples.samples_matrix
+        samples = samples.record.sample
     else:
-        samples = np.matrix(samples)
+        samples = np.asarray(samples, dtype=np.int8)
```

Comment:
Did you intentionally use np.asarray rather than np.array here? I expect you did but just making sure.